### PR TITLE
Recent Logs Tag Overflow Bug

### PR DESCRIPTION
**Pull request by emilyliublair**
## Recent Logs Tag Overflow Bug

Issue Number(s): #109 

What does this PR change and why?

Changes FE design of recent logs display on the dashboard to match Figma design. 3 most recent unique tags are displayed and all other remaining unique tags are displayed on hover. 

### Checklist

- [ ] Requirements have been implemented
- [ ] Acceptance criteria is met
- [ ] Database schema [docs](https://www.notion.so/gtbitsofgood/Database-Schema-Docs-a841a38d02db4428a7cdc85dcbb8a35c?pvs=4) have been updated or are not necessary
- [ ] Relevant reviewers (EM) have been assigned to this PR

### Critical Changes

N/A

### Related PRs

PR Number(s): N/A

### How To Test

Add tags to dog and hover over "+_" to see hover component.

### CODE REVIEW

**Comment by dograh** [comment]
The design looks good-- great job!

File: src/components/Hover.jsx
Comment (afazio1) [inline src/components/Hover.jsx:1]: can we rename this to something like `RecentTags.jsx`

**Review by afazio1 — APPROVED** [review]
Awesome! Was able to fix the bug with the filter tags not working

**Review by laurenmcp — APPROVED** [review]
Nice job Emily!

**Pull request merged**